templates/mrepo.conf.erb

```diff
@@ -18,6 +18,12 @@ rhnlogin = <%= @rhn_username %>:<%= @rhn_password %>
 rhnlogin = <%= @rhn_username %>
 <% end -%>
 
-<% unless (mailto = @mailto) == 'UNSET' -%>
-mailto = <%= mailto %>
+<% if @mailto -%>
+mailto = <%= @mailto %>
+<%   if @mailfrom -%>
```

Comment:
Some indentation wonkiness here? 

`<%   if @mailfrom -%>`
to
`<% if @mailfrom -%>`